SampleApplications/Samples/XamarinClient/XamarinClient/XamarinClient.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
>netstandard2.0< [](start = 21, length = 15)

interesting, so xamarin doesn't require mono (net46) anymore?

Reply:
no, you can create a project that just use .netstandard